internal/object/interface.go

```diff
@@ -18,11 +33,4 @@ type ObjectClient interface {
 		Retrieves an object for the data store
 	*/
 	GetObject(cid string) (map[string]interface{}, error)
-
```

Comment:
was assert removed from the interface?

Reply:
It didn't look like it was being used, is it necessary?

Reply:
it was intended for the motor to wrap in order to provide a static assert endpoint for objects. it will build the object and assert on it and return an error if it doesnt comply with schemas. I should add some documentation so that it was more obvious.

Reply:
If it's a helper method then the interface doesn't necessarily need it because other implementations may do things differently.